### DISCUSSION TITLE
Reset password - security feature request

### DISCUSSION THREAD

**Discussion by silavjy** [ovh/manager#5089] [Ideas]
**AS IS**

The current procedure to reset a password:

1. From the account details we click on the security tab and then click on the change button next to the password item:
![image](https://user-images.githubusercontent.com/67747632/121330318-2f012600-c916-11eb-8a1b-634656b9efbd.png)

2. A popup appears explaining that an email with instructions is to be sent to the email address for the account to be updated:
![image](https://user-images.githubusercontent.com/67747632/121330594-712a6780-c916-11eb-9921-554d3027a736.png)

3. An email is received with a link to update the password:
![image](https://user-images.githubusercontent.com/67747632/121330948-bea6d480-c916-11eb-86b9-72aa0a0b321a.png)

4. On clicking the link, a page opens with the following form:
![image](https://user-images.githubusercontent.com/67747632/121331131-e26a1a80-c916-11eb-8e1b-c3397e7ad63e.png)

**Problem statement:** The issue is that the password can be changed by a person other than the user in case someone else gets hold of that email (specially if the account doesn't have 2FA activated). **This is a security risk.**

**TO BE**

On step 4 it is suggested to request the use to enter the old password prior changing the credentials.
![image](https://user-images.githubusercontent.com/67747632/121332579-4b9e5d80-c918-11eb-8a6c-44338e73debb.png)

**Comment by antleblanc** [comment]
Hi @silavjy!

Thanks for reporting us this feature request.

I will report it to the team and we will keep you posted about it.

Thank you!

cc @Alkorin
